src/client/dfs/dfuse_hl.c

```diff
@@ -1,5 +1,5 @@
 /**
- * (C) Copyright 2018 Intel Corporation.
+ * (C) Copyright 2018Copyright 2018-2019 Intel Corporation.
```

Comment:
Looks like an editing issue here.

Reply:
Yeah.  Files are getting edited without having copyrights updated and so when I merge, my git hook tries to update them.  It shouldn't on merges, and I have remedied that, but people need to be updating those.

We ought to add the check for that in our checkpatch/linting step.